projects/core/src/lib/util/components/error-page/backend-status-page.component.html

```diff
@@ -0,0 +1,22 @@
+<div *ngIf="backendInfo">
+    <h2>Backend info:</h2>
+
+    <dl>
+        <dt>Backend version:</dt>
+        <dd>{{ backendInfo.version }}</dd>
+        <dt>Backend features:</dt>
+        <dd>{{ backendInfo.features }}</dd>
+        <dt>Build date:</dt>
+        <dd>{{ backendInfo.buildDate }}</dd>
+        <dt>Commit hash:</dt>
+        <dd>{{ backendInfo.commitHash }}</dd>
+    </dl>
+</div>
+
+<div *ngIf="error">
+    <h2>There was an error ...</h2>
```

Comment:
Say something like: The backend couldn't be reached.

Reply:
part of [99a0802](https://github.com/geo-engine/geoengine-ui/pull/512/commits/99a080258fa877d66157cf1184739c316b901dc2)